tmt/steps/provision/podman.py

```diff
@@ -151,6 +174,21 @@ def start(self):
             ['--name', self.container, '-v', f'{workdir}:{workdir}:Z',
              '-itd', self.image])[0].strip()
 
+    def reboot(self, hard=False, container_name=None):
+        if not self.podman:
+            return
+
+        container_name_win = container_name
+        if self.container:
+            container_name_win = self.container
```

Comment:
What about simplifying this part a bit?

```suggestion
            container_name = container_name or self.container
```

Hmmm, btw, is the `container_name` needed? I'd say the guest object should always take care of itself only (and it should always know its name).

Reply:
It should but I don't think is known...

def load(self, data):
....
        # Instances variables initialized later
        self.container_id = None

def start(self):
...
     self.container = 'tmt' + re.sub('[/ ]', '-', workdir)

I selected container name but maybe I should select container_id...
However it doesn't change situation much imho. Because start needs to be called but maybe I'm wrong.